kuzzle-sdk-java/doc/3/protocols/websocket/constructor/index.md

```diff
@@ -0,0 +1,43 @@
+---
+code: true
+type: page
+title: constructor
+description: This constructor creates a new WebSocket connection, using the specified options.
+---
+
+# constructor
+
+This constructor creates a new WebSocket connection, using the specified options.
+
+## Arguments
+
+```java
+public WebSocket(String host) throws URISyntaxException, IllegalArgumentException;
```

Comment:
(optional / debate) 

Since Java features a [URL class](https://docs.oracle.com/en/java/javase/13/docs/api/java.base/java/net/URL.html), why don't we use it,  as done with our C# SDK?

It seems simpler to use for users.

Reply:
I think this could be a good idea, I will implement it as an overload of the constructor

Reply:
Done [634549342d34a1d22e919eed410363565a691a27]
I'm using `URI` instead of `URL` because URL doesn't support `ws` and `wss` protocol